src/views/reset.hbs

```diff
@@ -1,16 +1,18 @@
 {{> header }}
 
 <div class="background-image cover aspect-ratio--object">
-  <div class="main-background-overlay cover aspect-ratio--object ">
```

Comment:
@dangerdak Isn't the 'main-background-overlay' the css class for having the background overlay in the reset page??